src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -17,7 +17,11 @@ var _trans_icon = '';
 var _trans_link = '';
 var _trans_password = '';
 var _trans_drag = '';
-
+const tableEditorBeforeSaveEvent = new CustomEvent('tableEditorBeforeSaveEvent', {
+    detail: {
+        description: 'is dispatched in ExecutePostCommand and SaveViaAjax and Save'
+    }
```

Comment:
Usually, the property "details" is used for the custom parameters assigned per event instance, but this is a classic comment (may be use `// ...` ahead this instantiation instead)
```suggestion
```